homework_4/bet_acceptance_api/internal/api/controllers/betValidator.go

```diff
@@ -0,0 +1,7 @@
+package controllers
+
+import "github.com/superbet-group/code-cadets-2021/homework_4/bet_accceptance_api/internal/api/controllers/models"
+
+type BetValidator interface {
+	IsBetValid(betRequest models.BetRequestDto) bool
```

Comment:
A validator can also return an error and you can return an error to the customer stating which argument is not valid and why.

Reply:
Feel free to change the code to have more descriptive validation error